master/buildbot/steps/package/rpm/rpmbuild.py

```diff
@@ -88,6 +94,9 @@ def __init__(self,
             'stdio', logobserver.LineConsumerLogObserver(self.logConsumer))
 
     def start(self):
+
+        self.rpmbuild = ('%s --define "dist %s"' % (self.rpmbuild, self.dist))
```

Comment:
@djmitche That was easy enough to fix, but this pattern repeats itself in the code already. Would it make sense to covert it to a dict instead and then generate command based on that ?